sw/device/silicon_creator/lib/drivers/watchdog_functest.c

```diff
@@ -24,7 +25,7 @@
 // Tests that we can pet the watchdog and avoid a reset.
 static rom_error_t watchdog_pet_test(void) {
   // Set watchdog bite threshold to 5ms.
-  uint32_t threshold = 5 * (kClockFreqAonHz / 1000);
+  uint32_t threshold = 5 * udiv64_slow(kClockFreqAonHz, 1000, NULL);
```

Comment:
This is somewhat curious. Does this somehow get optimized out, maybe with LTO? If not, do we expect `kClockFreqAonHz` to change at run time, and could we avoid spending instructions on calculating a constant?

Reply:
LTO should optimize it out, but it is currently not turned on. Mike Munday has been looking into LTO before.

As it was today, it was definitely _not_ optimized out, and instead producing a call to `__udivdi3`, a 1KB polyfill.